site/docs/_shared/CriminalPluginsTable.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { PLUGINS } from './data/plugins';
+
+const CriminalPluginsTable = () => {
+  // Filter for plugins with criminal vulnerability type
+  const criminalPlugins = PLUGINS.filter((plugin) => plugin.vulnerabilityType === 'criminal');
```

Comment:
Instead of creating a table for each `vulnerabilityType`, let's give `PluginTable` a prop that filters by `vulnerabilityType` if set.